firstvoices/backend/serializers/fields.py

```diff
@@ -27,7 +27,10 @@ def get_url(self, obj, view_name, request, format):
         kwargs = {self.lookup_url_kwarg: lookup_value}
 
         # add the site lookup
-        kwargs.update({"site_slug": get_site_from_context(self).slug})
+        if get_site_from_context(self):
```

Comment:
check If the site_slug is not present in the url, such as in the case of search API